### ISSUE TITLE
MobileAccessibility doesn't recognize installed Cordova plugin

### ISSUE THREAD

**Issue by janpio** [ionic-team/ionic-native#1860]
**I'm submitting a ...** 
[x] bug report
[ ] feature request

**Current behavior:**
Even if you install the MobileAccessibility by the docs it doesn't work on devices. It just doesn't recognize the installed (and working if used directly) Cordova plugin. You always get:

![image](https://user-images.githubusercontent.com/183673/28924408-f87d2402-7861-11e7-83f8-87e66e869098.png)

My guess: The `pluginRef` with `MobileAccessibility` is wrong and should be `window.MobileAccessibility`.

(I tried to fix this directly in the .js of the installed @ionic-native plugin, but that didn't work - so I am not sure and couldn't test this.)

**Expected behavior:**
Plugin works.

**Steps to reproduce:**
A project with both Ionic Native and direct access to the Cordova plugin is here: https://github.com/janpio/ionic-native-mobile-accessibility

**Related code:**

doesn't work:
```
     this.mobileAccessibility.getTextZoomCallback().then(res => {
        console.log('his.mobileAccessibility.getTextZoomCallback ', JSON.stringify(res));
      });
```

does work:
```
      function getTextZoomCallback(textZoom) {
        console.log('Current text zoom = ' + textZoom + '%')
      }
      window.MobileAccessibility.getTextZoom(getTextZoomCallback);
```

**Comment by KiwiKilian** [comment]
Can confirm this problem. When using `window.MobileAcessiblity` it does give me correct results in iOS Simulator, but also throwing an error `Property 'MobileAccessibility' does not exist on type 'Window'.`.
